dice_rolling_simulator.py

```diff
@@ -0,0 +1,96 @@
+#Made on May 27th, 2017
+#Made by SlimxShadyx
+
+#Dice Rolling Simulator
+
+import random
+
+#These variables are used for user input and while loop checking.
+correct_word = False
+dice_checker = False
+dicer = False
+roller_loop = False
+
+#Checking the user input to start the program.
+while correct_word == False:
+
+    user_input_raw = raw_input("\r\nWelcome to the Dice Rolling Simulator! We currently support 6, 8, and 12 sided die! Type [start] to begin!\r\n?>")
```

Comment:
The prompt string is too long for one line. You may have a variable named prompt_str, and use () to limit characters number.